lib/shopify_app/configuration.rb

```diff
@@ -26,6 +28,14 @@ def has_webhooks?
     def has_scripttags?
       scripttags.present?
     end
+
+    def has_fulfillment_services?
+      fulfillment_services.present?
+    end
+
+    def has_base_url?
+      base_url.present?
+    end
```

Comment:
How do you want the interface to look then?

My use-case on heroku like hosting I want to set a `ENV['CURRENT_APP_URL']` and forget about it, so my webhooks look something like this:

```
config.webhooks = [
  {topic: 'orders/create', address: File.join(ENV['CURRENT_APP_URL'], 'webhooks/orders_create'), format: 'json'}
]
```

And I'm doing something similar here: https://github.com/Shopify/shopify_app/pull/270/files#diff-586b77bc2e3ea60fba8a578c84145c02R56

This means the initializer interface doesn't care about the `callback_url`, it's automatically namespaced and set correctly. What it doesn't cover is letting ShopifyApp's parent act only as a means to set the `callback_url` and not respond to it.

How do you want it to look?